packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFDataConverters.m

```diff
@@ -192,9 +192,16 @@ WKNavigationActionPolicy FWFNativeWKNavigationActionPolicyFromEnumData(
 }
 
 FWFNSErrorData *FWFNSErrorDataFromNativeNSError(NSError *error) {
-  return [FWFNSErrorData makeWithCode:@(error.code)
-                               domain:error.domain
-                 localizedDescription:error.localizedDescription];
+  NSMutableDictionary *userInfo;
+  if (error.userInfo) {
+    userInfo = [NSMutableDictionary dictionary];
+    for (NSErrorUserInfoKey key in error.userInfo.allKeys) {
+      if ([error.userInfo[key] isKindOfClass:[NSString class]]) {
+        userInfo[key] = error.userInfo[key];
```

Comment:
Optional: I think it would be slightly clearer if you extracted a local `NSObject *value = error.userInfo[key]`  before the `if` and used that in both places, so it's extremely clear that the thing being checked and the thing being saved are in fact the same object (since if they were out of sync it could potentially crash).